app/controllers/users_controller.rb

```diff
@@ -163,6 +163,10 @@ def onboarding_update
   end
 
   def onboarding_checkbox_update
+    # TODO: mstruve will remove once debugging is done
+    Rails.logger.error("onboarding_checkbox_update_params:#{params}")
+    Rails.logger.error("onboarding_checkbox_update_user_params:#{params[:user]}")
```

Comment:
Anything else I should add or look at? I'm pretty perplexed right now so this seemed like a good start